src/App.tsx

```diff
@@ -1,34 +1,47 @@
 /* eslint-disable max-len */
-import React from 'react';
+import { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import { getTodos } from './api';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
-import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [sortedTodos, setSortedTodos] = useState<Todo[]>(todos);
+  const [isLoaded, setIsLoaded] = useState(false);
+
+  useEffect(() => {
+    getTodos()
+      .then(json => {
+        setTodos(json);
+        setSortedTodos(json);
+        setIsLoaded(true);
+      });
```

Comment:
Let’s use async await. It gives you a simpler way to work with asynchronous promise-based code